documentation/docs/guide/SiteEditor/Patterns/DSWPCardWithHyperLinkList.md

```diff
@@ -0,0 +1,84 @@
+# DSWP Card with Hyperlink List
+
+This card-based layout pattern is ideal for presenting multiple related items, such as services, features, or categories, in an organized grid format. It provides users with clear navigation options through linked lists while maintaining visual consistency across your WordPress site.
+
+> **Important**: Each pattern consists of several blocks. It's crucial to only adjust the content and color block settings, as the structural settings like padding and margins are designed to ensure the pattern remains responsive. Altering the structural block settings may cause the pattern to function improperly.
```

Comment:
blockquote or would something like this work better ::: warning (vuepress markdown)
The colour contrast might be improved in future for the blockquote styles 
![image](https://github.com/user-attachments/assets/0903ee10-b804-473c-b67d-acc5d5e87e4a)